app/serializers/site_serializer.rb

```diff
@@ -21,5 +21,6 @@ class SiteSerializer < ActiveModel::Serializer
   )
 
   has_many :categories
+  has_many :serials
```

Comment:
This includes serials data to site API's response.
However it is unnecessary for site.